### PR TITLE
Add examples for each feature + minor edits

### PR DESCRIPTION
**Pull request by sharmaine1028**
Status: completed

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
+Well, hello there! Welcome to your personal budgeting assistant, PennyWise. After this guide, you will understand how to
+use PennyWise, and spend every penny wisely. 🤑
 
 ## Introduction
 
-PennyWise is a desktop application that **empowers students with the ability to make informed financial decisions**,
-by providing a **graphical analysis of their financial activities**.
-It provides a clean Graphical User Interface (GUI) for easy comprehension of expenditure and savings.
+PennyWise is a desktop application that **empowers students with the ability to make informed financial decisions**, by
+providing a **graphical analysis of their financial activities**. It provides a clean Graphical User Interface (GUI) for
```
Comment (JasonYapzx) [inline docs/UserGuide.md:11]: thanks for adding this, i missed out this GUI definition

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,75 +1,77 @@
...
+easy comprehension of expenditure and savings.
 _Interested?_ Head over to [Quick Start](#quick-start) to get started. Enjoy!
 
 <div markdown="span" class="alert alert-info">:information_source: **Info:**
-Already an expert? Jump right straight to the [features](#features) section to see how you can get the most out of PennyWise!
+Already set PennyWise up? Jump right straight to the [features](#features) section to see what features are available!
```
Comment (JasonYapzx) [inline docs/UserGuide.md:16]:
```suggestion
Already done with the set-up? Jump right straight to the [features](#features) section to see what features are available!
```

Would this sound better

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1,75 +1,77 @@
...
 
+<!-- TOC -->
 
 ## Purpose of Guide
-This guide aims to teach those entirely new to PennyWise how to navigate and utilise the application. It can also 
-provide experienced users or users familiar with similar command-line applications tips on how to maximise efficiency
-when using PennyWise. 
+
+This guide aims to teach those new to PennyWise how to navigate and utilise the application. It also provides
+experienced or returning users a mini refresher course on how to make the most out of PennyWise.
```
Comment (JasonYapzx) [inline docs/UserGuide.md:65]:
```suggestion
This guide aims to teach those new to PennyWise how to navigate and utilise the application. It also acts as 
a refresher for any experienced or returning users, allowing you to make the most out of PennyWise.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -140,40 +147,50 @@ Curious about the various commands? Refer to the [Features](#features) below to
...
-| **Command Input Box**  |                                       where you enter your commands to use PennyWise                                       |
-| **Command Output Box** |                                  where you see the output of your commands from PennyWise                                  |
+|    **List
+Display**    |            where you view your entry lists (Expenditure or Income) <br> - can be modified by the `view` command            |
+|   **Graph
+Display**    | where you view your entries in a graphical overview (Pie Chart or Line Graph) <br> - can be modified by the `view` command |
+| **Command Input
+Box**  |                                       where you enter your commands to use PennyWise                                       |
+| **Command Output
+Box** |                                  where you see the output of your commands from PennyWise                                  |
```
Comment (JasonYapzx) [inline docs/UserGuide.md:164]: Not sure why the formatting for this is adjusted, looks a bit weird when viewed in markdown

image

File: docs/UserGuide.md [hunk 5]
```diff
@@ -481,17 +535,24 @@ without compromising on quality and user experience so do look out for our futur
...
+| **
+Summary**         | `summary [mo/MONTH]`<br> e.g. `summary mo/2022-09`                                                                                                      |
+| **View (
+Category)** | `view t/ENTRY_TYPE` <br> e.g. `view t/e`                                                                                                                |
+| **View (
+Month)**    | `view t/ENTRY_TYPE mo/MONTH` <br> e.g. `view t/i mo/2022-10`                                                                                            |
+| **
+Clear**           | `clear`                                                                                                                                                 |
+| **
+Exit**            | `exit`                                                                                                                                                  |
```
Comment (JasonYapzx) [inline docs/UserGuide.md:555]: Same issue with the table here

**Review by JasonYapzx — APPROVED** [review]
Thanks for the reformatting, just small nits to pick, otherwise LGTM!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -297,20 +321,26 @@ Edits an entry, where at least **1** of the optional fields description/amount/d
...
+
 - Example: `edit 2 t/i a/150.00 da/22-10-2022`
     - The income at the 2nd position will have its amount and date edited
 - Expected: `Edited Entry: Monthly Allowance; Date: 02-01-2022; Amount: 150.00; Tag: Allowance`
 
 ![EditCommandIncome](images/ug/EditCommandIncome.png)
 
 ### Deleting entries
+
+You were going to watch the latest Kungfu Panda movie, but your friend bailed on you :(. You already logged the entry so
```
Comment (EmilyOng) [inline docs/UserGuide.md:341]: Should we add an user story for deleting income too?

File: docs/UserGuide.md [hunk 7]
```diff
@@ -376,27 +412,35 @@ The command format is provided below:
...
 
 2. View a Pie Chart of all incomes by categories
-   - Examples: `view t/i`
-   - Expected: `Show graphically all income by category` and a Pie Chart on the right of the application
+    - Examples: `view t/i`
+    - Expected: `Show graphically all income by category` and a Pie Chart on the right of the application
 
 ### View entries by month
+
+You went on a shopping spree today. Find out how today's expenditure compared to the rest of the month.
```
Comment (EmilyOng) [inline docs/UserGuide.md:442]: Should we talk about income too?

File: docs/UserGuide.md [hunk 8]
```diff
@@ -425,17 +475,21 @@ Danger zone! This command is irreversible- it is not possible to retrieve entrie
 </div>
 
 ### Exiting the program
+
+You're done logging your expenses for the day. Let's exit the application.
```
Comment (EmilyOng) [inline docs/UserGuide.md:479]:
```suggestion
You're done logging your expenses and income for the day. Let's exit the application.
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -246,19 +268,16 @@ OR we could even [edit](#editing-entries) the original entry directly to reflect
...
-
 <p align="right">
     <a href="#top">Back to Top </a>
 </p>
 
 ## Features
 
 ### Adding entries
+
+Perhaps you just bought a cup of coffee, or a paycheck comes in. Add these entries to PennyWise to keep track of them.
```
Comment (EmilyOng) [inline docs/UserGuide.md:279]: It's nice!

File: docs/UserGuide.md (hunk 2 above)
Comment (sharmaine1028) [reply docs/UserGuide.md:16]: Yep thanks for the suggestion!

File: docs/UserGuide.md (hunk 6 above)
Comment (sharmaine1028) [reply docs/UserGuide.md]: Have added @EmilyOng, do help me see it is okay :D

File: docs/UserGuide.md (hunk 7 above)
Comment (sharmaine1028) [reply docs/UserGuide.md:478]: Have added @EmilyOng, do help me see it is okay :D

Comment (EmilyOng) [reply docs/UserGuide.md:478]: Looks good!

File: docs/UserGuide.md [hunk 10]
```diff
@@ -474,6 +477,9 @@ Format: `view t/ENTRY_TYPE`
 
 You went on a shopping spree today. Find out how today's expenditure compared to the rest of the month.
 
+You were tipped for your exceptional service as a part-time server today. Find out how today's income weighs up to the
```
Comment (EmilyOng) [inline docs/UserGuide.md:480]:
```suggestion
Or, were you tipped for your exceptional service as a part-time server today? Find out how today's income weighs up to the
```

Not sure if the sentence started a bit abruptly from the previous line

**Review by EmilyOng — APPROVED** [review]
Great!!

**Comment by JasonYapzx** [comment]
Lets resolve the conflicts and changes after class today

**Pull request merged**